CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contributing to Rig
+
+Thank you for considering contributing to Rig! Here are some guidelines to help you get started.
+
+## Issues
+
+Before reporting an issue, please check existing or similar issues that are currently tracked.
+
+## Pull Requests
+
+Contributions are always encouraged and welcome. Before creating a pull request, create a new issue that tracks that pull request describing the problem in more detail. Pull request descriptions should include information about it's implementation, especially if it makes changes to existing abstractions.
+
+PRs should be small and focused and should avoid interacting with multiple facets of the library. This may result in a larger PR being split into two or more smaller PRs. Commit messages should follow the [Conventional Commit](conventionalcommits.org/en/v1.0.0) format (prefixing with `feat`, `fix`, etc.) as this integrates into our auto-releases via a [release-plz](https://github.com/MarcoIeni/release-plz) Github action.
```

Comment:
Add an example or template for PR messages and Issues. This helps me and I'm sure will help others follow our best practices.

Reply:
@Tachikoma000 templates are provided when you create a new issue/PR so I don't think we need them here